### PR TITLE
Feature - TempHP as a character attribute

### LINKED ISSUE #22: Features should be able to add temporary hitpoints

### PR DESCRIPTION
**Pull request by Dumbgenius**
Resolves #22. 

This branch adds `TempHP` as a character attribute. Since it is an attribute, anything that can apply an effect to the character can add temporary hit points.

Old mentions of "Temporary Hit Points" on the health card have been renamed to "Extra Hit Points" to avoid confusion. However, the collection `TemporaryHitPoints` hasn't been renamed, since doing so would probably cause issues.

Since this branch adds a character attribute and thus modifies the Character schema, it requires a database migration.

### CODE REVIEW

File: rpg-docs/client/views/character/stats/healthCard/healthCard.css [hunk 1]
```diff
@@ -3,10 +3,27 @@
...
 
+.healthCard #temporaryHitPointSlider {
+	--paper-diff-slider-active-color: #009688; /* teal */
+	--paper-diff-slider-knob-color: #009688; /* these don't seem to work? */
+	--paper-diff-slider-pin-color: #009688;
+}
+
+.healthCard .extraHitPointSlider {
+	--paper-diff-slider-active-color: #00BCD4; /* cyan */
+	--paper-diff-slider-knob-color: #00BCD4; /* these don't seem to work? */
```
Comment (Dumbgenius) [inline rpg-docs/client/views/character/stats/healthCard/healthCard.css]: I took these property names from `paper-diff-slider.html`, but they don't seem to work.

Comment (ThaumRystra) [inline rpg-docs/client/views/character/stats/healthCard/healthCard.css]: So you can't use custom css attributes (that start with `--`) in css compiled by Meteor. You need to put styles that use custom css attributes in `public > custom_components > app-theme.html`

Comment (Dumbgenius) [inline rpg-docs/client/views/character/stats/healthCard/healthCard.css]: Thanks, I didn't know that.

Comment (ThaumRystra) [reply rpg-docs/client/views/character/stats/healthCard/healthCard.css]: Fixed this now

**Pull request merged**